### ISSUE TITLE
Term request: mass spectrometry imaging

### ISSUE THREAD

**Issue by PaulVanSchayck** [openrif/vivo-isf-ontology#738]
I would like to submit a term request for the technique "mass spectrometry imaging" (MSI).

This technique ([wikipedia description](https://en.wikipedia.org/wiki/Mass_spectrometry_imaging)) is in essence the same as mass spectrometry, but then repeated in a pixelated way across for example a tissue slide.

I'm not sure how it would fit best, as it's an imaging assay. But many (if not most, not MSI expert) MS assays like MALDI also can be applied in MSI. So some of the sub assays of MS can also fit under MSI.

This is the Mass Spectrometry Imaging term in the MS ontology:
http://purl.obolibrary.org/obo/MS_1002521

**Comment by pitviper6** [comment]
@marijane 

I looked this up and the term has a namespace in Ontobee:
http://purl.obolibrary.org/obo/MS_1002521

I'm not exactly sure where it would fit in the eagle-i ontology. In Ontobee, it's an object attribute, part of the class spectrum generation information.  It's not part of the image technique or imaging assay sections.

http://www.ontobee.org/ontology/ERO?iri=http://purl.obolibrary.org/obo/ERO_0001312
http://www.ontobee.org/ontology/ERO?iri=http://purl.obolibrary.org/obo/OBI_0000185

Since we didn't capture the spectrum generation information class, would it be best to add that entire class and its subclasses, or add the class and just the one subclass, or punt this one and tell them that we can't add it at this time?

Juliane.

**Comment by marijane** [comment]
Hmmmmm.

I don't think we should add the entire Spectrum Generation Information class hierarchy from the Mass Spectrometry Ontology into eagle-i.  The MS ontology is a very detailed ontology (almost 6000 classes) designed for the very specific purpose of annotating mass spectrometry experiments.  I think this level of detail may go beyond the scope of eagle-i.  It does not appear to use BFO, and most of the subclasses of the spectrum generation information class are defined as anonymous class restrictions like this:
```
<rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">object attribute</rdfs:label>
    <rdfs:subClassOf>
        <Restriction>
            <onProperty rdf:resource="http://purl.obolibrary.org/obo/ms#part_of"/>
            <someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/MS_1001458"/>
       </Restriction>
</rdfs:subClassOf>
```
As such, I think it would be difficult to figure out how to make this structure fit into eagle-i.  This structure may also complicate extraction (I could only extract direct subclasses with ROBOT, though there may be another way to extract anonymous restriction classes) and it obscures how many classes we'd be adding, which may be quite numerous. 

Adding just the MSI class seems fine, though I'm not sure we want to borrow the URI from the MS ontology unless eagle-i is already borrowing terms from it, which I don't think it does.

This still leaves the question of where it should go in eagle-i.  I can see how putting it under imaging assay makes sense, but given that MALDI TOF already exists in eagle-i under the mass spectrometry class, and given that we can't introduce a polyhierarchy for terms that seem to fit in more than one place, i'm inclined to put it there with the rest of the mass spectrometry subclasses.  Also, in the eagle-i reference taxonomy, mass spectrometry is a molecular assay, and given both eagle-i's definition for molecular assay and the wikipedia definition for MSI, this doesn't seem like an inappropriate place to put it.  I'm going to tag @nicolevasilevsky to possibly weigh in on this, since she was the term editor for many of the assay classes in eagle-i, perhaps she can shed some light here...?

TL;DR:
- just add the one class
- consider minting a new ERO URI for it
- make it a subclass of mass spectrometry to group it with the rest of the mass spectrometry subclasses

**Comment by nicolevasilevsky** [comment]
I agree with you @marijane, I think it should be a subclass of mass spectrometry assay.
